project/sites/ircommission/themes/ircommission_theme/src/scss/components.scss

```
@@ -662,11 +662,11 @@ position:static;
   list-style: none;
 }
 
-.js-view-dom-id-437fbea96a99d3c9824ab36a7f532f40dec1fb5d0160fd93a71334916a142201 header {
+.block-views-blockfront-latest-news-home-latest-newsheader .content header {
```

There is nothing necessarily wrong with writing it this way, however, with using SCSS, you can reduce code duplication by writing it out like this:

```suggestion
.block-views-blockfront-latest-news-home-latest-newsheader,
.block-views-blockirc-key-documents-views-block-key-docs-homepage {
   .content header {
      text-align: center;
   }
}
```